lineage/figures/figure9.py

```diff
@@ -20,59 +20,58 @@ def makeFigure():
     lapatinib = [Lapatinib_Control + Gemcitabine_Control, Lapt25uM, Lapt50uM, Lap250uM]
     gemcitabine = [Lapatinib_Control + Gemcitabine_Control, Gem5uM, Gem10uM, Gem30uM]
 
-    def find_AIC(data, desired_num_states):
+    def find_AIC(data, desired_num_states, num_cells):
         # Copy out data to full set
         dataFull = []
         for _ in desired_num_states:
             dataFull.append(data)
 
         # Run fitting
         output = run_Analyze_over(dataFull, desired_num_states, atonce=True)
-        AICs = np.array([oo[0][0].get_AIC(oo[2], atonce=True)[0] for oo in output])
+        AICs = np.array([oo[0][0].get_AIC(oo[2], num_cells, atonce=True)[0] for oo in output])
 
         return AICs - np.min(AICs, axis=0)
 
-    lapAIC = find_AIC(lapatinib, desired_num_states)
-    gemAIC = find_AIC(gemcitabine, desired_num_states)
+    lapAIC = find_AIC(lapatinib, desired_num_states, num_cells=5290)
```

Comment:
What is the numcells argument here for?

Reply:
The BIC needs the number of data points, I printed out the number of cells in both drug treatments and placed it here. I didn't change the name of the function, but it is now based on BIC.